src/lib/utilities/form/create-spa-form.test.ts

```diff
@@ -0,0 +1,134 @@
+// @vitest-environment jsdom
```

Comment:
This test feels pointless. is there a better way to unit test svelte components?